portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -5,14 +5,25 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
```

Comment:
Please sort per https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

Reply:
I think I reordered them correctly but if not let me know! Thank you!